### DISCUSSION TITLE
Remove terminal output when adding or deleting todos

### DISCUSSION THREAD

**Discussion by fritzrehde** [todotxt/todo.txt-cli#364] [Q&A] [answered]
Is there a way (maybe an option I have overlooked) to disable the terminal output l when adding or deleting todos? For example, when I type `todo.sh add "todo message"` I get the following output printed to the terminal:

1 todo message\
TODO: 1 added.

I find this information rather unnecessary and would like to not have it printed.

**Comment by inkarkat** [comment, marked as answer]
You can put

    export TODOTXT_VERBOSE=0

into your `todo.cfg` for a permanent configuration, or use `TODOTXT_VERBOSE=0 todo.sh ...` for one-time use.

(There's the `-v` command-line option to _increase_ the verbosity (from the default 1 to 2), but no corresponding option to _decrease_ it, so the full configuration variable has to be used. If you've silenced these messages as described, you can get back the messages via `-v`, as that then increases the level from 0 to 1 again.)

**Reply by fritzrehde** [reply]
Thank you very much! Do you think it would be a good idea to add this setting to the default config file, so new users will easily find this setting too?

**Reply by inkarkat** [reply]
Yes, I think adding this to the default configuration is a good idea, to make this easier to find and to document the possible values.

[I've created a pull request](https://github.com/todotxt/todo.txt-cli/pull/365).
